README.md

```diff
@@ -1,3 +1,5 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
+
```

Comment:
OK for now, but should we consider moving this into a file that is included by all pages as in https://stackoverflow.com/a/46765337